### ISSUE TITLE
Authors and Affiliations are not correctly interpreted

### ISSUE THREAD

**Issue by dleutnant** [dleutnant/rticles#1]
- [ ] Are $ in template.tex to indicate inline math allowed?
- [ ] How about one affiliation per multiple authors and vice versa?

header should be:
<img width="693" alt="bildschirmfoto 2017-12-21 um 12 56 11" src="https://user-images.githubusercontent.com/13165811/34254886-6d94a670-e64e-11e7-9414-ab4f4b3dc52e.png">

**Comment by dleutnant** [comment]
source:

```tex
% Author Orchid ID: enter ID or remove command
\newcommand{\orcidauthorA}{0000-0000-000-000X} % Add \orcidA{} behind the author's name
%\newcommand{\orcidauthorB}{0000-0000-000-000X} % Add \orcidB{} behind the author's name

% Authors, for the paper (add full first names)
\Author{Firstname Lastname $^{1,\dagger,\ddagger}$\orcidA{}, Firstname Lastname $^{1,\ddagger}$ and Firstname Lastname $^{2,}$*}

% Authors, for metadata in PDF
\AuthorNames{Firstname Lastname, Firstname Lastname and Firstname Lastname}

% Affiliations / Addresses (Add [1] after \address if there is only one affiliation.)
\address{%
$^{1}$ \quad Affiliation 1; e-mail@e-mail.com\\
$^{2}$ \quad Affiliation 2; e-mail@e-mail.com}
```

gives the following error:

```r
"template" (line 69, column 29):
unexpected "^"
expecting letter
Fehler: pandoc document conversion failed with error 5
```

which basically is `\Author{Firstname Lastname $^`

**Comment by dleutnant** [comment]
solved in 0f49517a6e66c45d3ec18dd397694fadf3e0a918
